docs/testing.md

```diff
@@ -139,37 +139,53 @@ The following steps are necessary to have a proper environment setup, where all
     Kubernetes cluster.
 
 
-3. Export the `CF_OPERATOR_WEBHOOK_SERVICE_HOST` env variable
+1. Export the `CF_OPERATOR_WEBHOOK_SERVICE_HOST` env variable
 
     ```bash
-    export CF_OPERATOR_WEBHOOK_SERVICE_HOST=$(minikube ssh -- "cat /etc/resolv.conf | grep nameserver | awk '{ printf \$2 }'")
+    export CF_OPERATOR_WEBHOOK_SERVICE_HOST=$(minikube ip)
     ```
 
     _**Note**_: You can also find the correct IP, by running `ip addr`. The IP address under `vboxnet1` is the IP that you need.
 
-4. Export the `OPERATOR_TEST_STORAGE_CLASS` env variable
+1. Export the `OPERATOR_TEST_STORAGE_CLASS` env variable
 
     ```bash
     export OPERATOR_TEST_STORAGE_CLASS=standard
     ```
 
     _**Note**_: Require for the PVC test creation, in minikube.
 
-5. Ensure `GO111MODULE` is set
+1. Copy the ssh-key required for tunneling to /tmp
```

Comment:
This is because `minikube ip` is used above? I don't set up any tunneling on Linux, but  export my public interface ip instead:

```
export CF_OPERATOR_WEBHOOK_SERVICE_HOST=$(ip -4 a s dev `ip r l 0/0 | cut -f5 -d' '` | grep -oP 'inet \K\S+(?=/)')
```

Reply:
I went for the tunneling, because I saw that it is already part of the integration test setup (https://github.com/cloudfoundry-incubator/cf-operator/blob/d053b0eca5dfb07f83657fdd9ee85c74a269c7d5/integration/environment/operator.go#L61).
If `ssh_server_user` is set, the test framework expects the ssh-key of the node to be at `/tmp/cf-operator-tunnel-identity`.

Reply:
Ah, we only use the tunnel because we have to, there is no direct connection possible between the CI container and the kube cluster. I don't use it when testing locally.

However that's very interesting, can you put it in a separate section, or mention it's optional?

Reply:
I just used the tunnel, because exporting the IP you get with `minikube ssh -- "cat /etc/resolv.conf | grep nameserver | awk '{ printf \$2 }'` didn't work for me. I just tested it with the correct IP and it's way simpler. I will adjust the doc.